src/tigerbeetle/main.zig

```diff
@@ -422,6 +469,9 @@ const Command = struct {
                     @field(config.process, field_name),
                 );
             }
+
+            try stdout.writeAll("\n");
+            try vsr.multiversioning.print_information(allocator, stdout);
```

Comment:
```suggestion
            if (builtin.target.os.tag == .linux) {
                try stdout.writeAll("\n");
                try vsr.multiversioning.print_information(allocator, stdout);
            }
```

...Otherwise windows/mac will panic on `tigerbeetle version --verbose

Reply:
Oops - I've gone a slightly different route, it'll print out it's not supported.